### PR TITLE
Fixing Issue #31348

### LINKED ISSUE #31348: Missing VSCode Image
### Description

At the section: "Update the /Views/Movies/Create.cshtml with a Rating field",
The Visual Studio Code tab is empty, instead of have an image showing the updated form after inserting the new field "Rating".
Issue

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/new-field?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-mvc-app/new-field.md

### Document ID

c3fb9673-4da6-5eeb-08bb-6f8e7bc8e0c6

### Article author

@wadepickett

### PR DESCRIPTION
**Pull request by Nitzantomer1998**
Fixes #31348

Changed two files:
+ Inserted a new image which show how to update the existed form with the new "Rating" field in VScode.
Path of the new file: aspnetcore/tutorials/first-mvc-app/new-field/_static/VSCode-UpdatingForm.png

+ Inserted a new row inside of the visual studio code tab, which shows the needed action in VSCode.
Path of the modified file: aspnetcore/tutorials/first-mvc-app/new-field.md

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-mvc-app/new-field.md](https://github.com/dotnet/AspNetCore.Docs/blob/fa83127ed4f4563f8d0ac7ebe3c688e23b590d88/aspnetcore/tutorials/first-mvc-app/new-field.md) | [Part 8, add a new field to an ASP.NET Core MVC app](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/new-field?branch=pr-en-us-31349) |

### CODE REVIEW

**Review by wadepickett — CHANGES_REQUESTED** [review]
Looks great!  Thanks!   Could you add the same update to the .NET 7.0 version of the topic here as well?  (Or I can add it if you like):
aspnetcore/tutorials/first-mvc-app/new-field/includes/new-field7.md

**Comment by wadepickett** [comment]
@Nitzantomer1998, please disregard change request for version 7.0.   Looking back, the output for the class names were different for those elements in previous versions.   I will approve in a second with your original for version 8.0 and back out of the change I made for versions 5-6.

File: aspnetcore/tutorials/first-mvc-app/new-field/includes/new-field5.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ You can copy/paste the previous "form group" and let intelliSense help you updat
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-<!-- This tab intentionally left blank. -->
+![Updating the existed form by inserting the new field "Rating".](~/tutorials/first-mvc-app/new-field/_static/VSCode-UpdatingForm.png)
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-mvc-app/new-field/includes/new-field5.md:59]:
```suggestion
<!-- This tab intentionally left blank. -->
```

File: aspnetcore/tutorials/first-mvc-app/new-field/includes/new-field6.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ You can copy/paste the previous "form group" and let intelliSense help you updat
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-<!-- This tab intentionally left blank. -->
+![Updating the existed form by inserting the new field "Rating".](~/tutorials/first-mvc-app/new-field/_static/VSCode-UpdatingForm.png)
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-mvc-app/new-field/includes/new-field6.md:59]:
```suggestion
<!-- This tab intentionally left blank. -->
```

**Review by wadepickett — APPROVED** [review]
Thanks again!!

**Comment by Nitzantomer1998** [comment]
Hey man, I guess our country's timing is completely different.

I saw you made some changes to the older versions and then revert it if
i`m current.
If you need me to check the previous .NET versions and apply the same
change (with modification to the context) I would be happy to do so.

So if needed, let me know and I will do it tomorrow.
Have a good day and keep the good work :), those document really help me to
get started with .NET

**Pull request merged**